src/bin/am/commands/explore.rs

```diff
@@ -0,0 +1,69 @@
+use anyhow::Result;
+use autometrics_am::config::{endpoints_from_first_input, AmConfig, Endpoint};
+use autometrics_am::parser::endpoint_parser;
+use clap::Parser;
+use tracing::info;
+use url::Url;
+
+#[derive(Parser, Clone)]
+pub struct CliArguments {
+    /// The endpoint(s) that will be passed to Explorer
+    ///
+    /// Multiple endpoints can be specified by separating them with a space.
+    /// The endpoint can be provided in the following formats:
+    /// - `:3000`. Defaults to `http`, `localhost` and `/metrics`.
+    /// - `localhost:3000`. Defaults to `http`, and `/metrics`.
+    /// - `https://localhost:3000`. Defaults to `/metrics`.
+    /// - `https://localhost:3000/api/metrics`. No defaults.
+    #[clap(value_parser = endpoint_parser, verbatim_doc_comment)]
+    metrics_endpoints: Vec<Url>,
```

Comment:
The explore endpoint should accept a list of endpoint to scrape, but rather it should just accept a single prometheus endpoint